### ISSUE TITLE
Duplicate eight schools model

### ISSUE THREAD

**Issue by bbbales2** [MansMeg/posteriordb#139]
eight_schools and 8schools appear to be the same model

**Comment by bbbales2** [comment]
Other duplicates are:

arma and arma-arma11

pkpd and one_comp_mm_elim_abs-one_comp_mm_elim_abs

low_dim_gauss_mix_collapse and low_dim_gauss_mix_collapse-low_dim_gauss_mix_collapse

**Comment by MansMeg** [comment]
Yes. This is the case. There are posterior with aliases. (arma, pkpd) to simplify quick access to specific compute comparison models from Michaels repo. 

How would you like the API to work? Maybe have an include aliases in the names() function?

A quick fix is to only use the posteriors that have a "-". No alias has those dashes.

**Comment by MansMeg** [comment]
@bbbales2, do you have a take on this? What do you think would be the best way to handle alias so they dont end up as duplicates for you?

**Comment by bbbales2** [comment]
> How would you like the API to work? Maybe have an include aliases in the names() function?

I think the answer is the default way to iterate through all models shouldn't give duplicates.

And by default way I mean whatever I did, which maybe was bad lol. I vaguely remember I just tried to copy the tutorial.

So whatever the tutorial does just shouldn't give duplicates. If it already is that way and I just messed things up, I wouldn't be surprised.
